scripts/pull_and_compile_protos.py

```diff
@@ -1,22 +1,14 @@
 """
-SPDX-FileCopyrightText: Copyright (c) 2023 Contributors to the
-Eclipse Foundation
+SPDX-FileCopyrightText: 2024 Contributors to the Eclipse Foundation
```

Comment:
Not super important, but according to @stevenhartley and eclipse, the copyright date should be when the file was created, not edited. So, if this file was originally merged in 2023, copyright date should stay 2023.